02-introduction.rst

```diff
@@ -10,7 +10,8 @@ and your new friends (among others) are named "vectors", "arrays", "views" or
 Simple example
 --------------
 
-Let's take a very simple example, random walk.
+Let's take a very simple example, random walk. (You can execute any code below
+from the `code` folder, ideally inside an IPython session or Jupyter notebook.)
```

Comment:
Would `code`_  links to the repository ?

I don't think mentioning IPython or Jupyer here make sense here without further explaining what they are and this might not be the place.